packages/patternfly-4/react-core/src/components/Title/Title.tsx

```diff
@@ -21,7 +22,7 @@ export interface TitleProps extends Omit<React.HTMLProps<HTMLHeadingElement>, 's
   /** Additional classes added to the Title */
   className?: string;
   /** the heading level to use */
-  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  headingLevel?: headingLevelType;
```

Comment:
I think we need `headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';` for the docs.